docs/source/en/_toctree.yml

```
@@ -78,6 +78,8 @@
       title: Community pipelines
     - local: using-diffusers/contribute_pipeline
       title: How to contribute a community pipeline
+    - local: ddpo
+      title: Fine-tuning Stable Diffusion using Reinforcement Learning with DDPO
```

I think this would be more fitting in the **Training** section of the docs

```suggestion
      title: Reinforcement learning training with DDPO
```